### PR TITLE
Unzipping feature Added !

### PR DESCRIPTION
**Pull request by Rajchowdhury420**

### CODE REVIEW

File: tools/python/file_extract.py [hunk 1]
```diff
@@ -78,6 +78,10 @@ def hex_escape(s):
     return ''.join(escape(c) for c in s)
 
 
+
+with zipfile.ZipFile(path_to_zip_file, 'r') as zip_ref:
```
Comment (yuxuanchen1997) [inline tools/python/file_extract.py]: `path_to_zip_file` is not defined in this file, where does it come from?

File: tools/python/file_extract.py [hunk 2]
```diff
@@ -78,6 +78,10 @@ def hex_escape(s):
...
+    zip_ref.extractall(directory_to_extract_to)
```
Comment (yuxuanchen1997) [inline tools/python/file_extract.py]: same, what is `directory_to_extract_to`?

**Comment by Rajchowdhury420** [comment]
okay !

**Comment by Rajchowdhury420** [comment]
okay mate

**Pull request closed without merging**